packages/data-provider/README.md

```diff
@@ -0,0 +1,173 @@
+# Lyric (Data Provider)
+
+Package published on npm as https://www.npmjs.com/@overture-stack/lyric
```

Comment:
Could use a badge from shields.io . It uses the shields.io service to find and show the latest published npm version (and looks cool).
```md
[![NPM Version](https://img.shields.io/npm/v/@overture-stack/lyric?color=%23cb3837&style=for-the-badge&logo=npm)](https://www.npmjs.com/package/@overture-stack/lyric)
```
Heres a working example using the arranger-server repo:
[![NPM Version](https://img.shields.io/npm/v/@overture-stack/arranger-server?color=%23cb3837&style=for-the-badge&logo=npm)](https://www.npmjs.com/package/@overture-stack/arranger-server)